lib/internal/modules/esm/handle_process_exit.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+// Handle a Promise from running code that potentially does Top-Level Await.
+// In that case, it makes sense to set the exit code to a specific non-zero
+// value if the main code never finishes running.
+function handleProcessExit() {
+  process.exitCode ??= 13;
```

Comment:
This needs to be changed for Node.js 14.

Reply:
```suggestion
  if (process.exitCode == null) process.exitCode = 13;
```